src/quickstart/smart-contract.md

```diff
@@ -156,12 +174,23 @@ We now have an `out` directory that contains our build artifacts such as the JSO
 
 We will start by adding a Rust integration test harness using a Cargo generate template. If this is your first time going through this quickstart, you'll need to install the `cargo generate` command. In the future, you can skip this step as it will already be installed.
 
-Navigate to your contract and then run the installation command:
+Navigate to your contract folder:
```

Comment:
The user should already be in the contract folder at this step. We could move these commands to the "Build the Contract" section